spec/controllers/deliverers_controller_spec.rb

```diff
@@ -3,95 +3,83 @@
 RSpec.describe DeliverersController, type: :controller do
 
   # Create user for login
-  before(:each) do
+  Given {
     @user = FactoryGirl.create(:user)
     sign_in @user
-  end
+  }
 
   # Test new action
   describe "get #new" do
-    it "initializes a new deliverer" do
-      get :new
+    When { get :new }
 
-      expect(assigns(:deliverer)).to be_a_new(Deliverer)
-    end
+    Then { expect(assigns(:deliverer)).to be_a_new(Deliverer) }
   end
 
   # Test create action
   describe "post #create" do
 
     context "with valid attributes" do
-      it "creates new deliverer" do
-        expect{
-          post :create,
+      When {
+        post :create,
           params: { deliverer: FactoryGirl.attributes_for(:deliverer) },
           as: :json
-        }.to change{ Deliverer.count }.by(1)
+        }
 
-        is_expected.to redirect_to home_path
-      end
+      Then { expect(Deliverer.count).to eq 1 }
```

Comment:
We can also use natural assertions in Then clause. HB Backend follow that style.
https://github.com/jimweirich/rspec-given#natural-assertions
(again, just FYI)

Reply:
I'll take note of this too!